From f6492f679e78d58748171b731488f212b38e4821 Mon Sep 17 00:00:00 2001
From: Peter O'Connor <peter@solus-project.com>
Date: Sun, 3 Dec 2017 13:31:53 +1100
Subject: [PATCH 2/6] packages: Put straggling .pc files in devel package

Some sources utilise .pc files in /usr/share/pkgconfig/*.pc

This ensures that they are not included in the main package.

Signed-off-by: Peter O'Connor <peter@solus-project.com>
---
 ypkg2/packages.py | 1 +
 1 file changed, 1 insertion(+)

diff --git a/ypkg2/packages.py b/ypkg2/packages.py
index 02f23ce..d16de74 100644
--- a/ypkg2/packages.py
+++ b/ypkg2/packages.py
@@ -181,6 +181,7 @@ class PackageGenerator:
         self.add_pattern("/usr/lib/lib*.a", "devel")
         self.add_pattern("/usr/lib/pkgconfig/*.pc", "devel")
         self.add_pattern("/usr/lib64/pkgconfig/*.pc", "devel")
+        self.add_pattern("/usr/share/pkgconfig/*.pc", "devel")
         self.add_pattern("/usr/include/", "devel")
         self.add_pattern("/usr/share/man3/", "devel",
                          priority=PRIORITY_DEFAULT+1)
-- 
2.16.1

